build/build-rector-scoped.sh

```diff
@@ -34,18 +31,12 @@ rm -f "$BUILD_DIRECTORY/phpstan-for-rector.neon"
 
 # 2. scope it
 note "Running scoper to $RESULT_DIRECTORY"
-wget https://github.com/humbug/php-scoper/releases/download/0.14.0/php-scoper.phar -N --no-verbose
+wget https://github.com/humbug/php-scoper/releases/download/0.17.2/php-scoper.phar -N --no-verbose
 
 # Work around possible PHP memory limits
+php -d memory_limit=-1 php-scoper.phar add-prefix bin config src packages rules vendor composer.json --output-dir "../$RESULT_DIRECTORY" --config scoper.php --force --ansi --working-dir "$BUILD_DIRECTORY";
 
-if test -z ${PHP80_BIN_PATH+y}; then
-    php -d memory_limit=-1 php-scoper.phar add-prefix bin config src packages rules vendor composer.json --output-dir "../$RESULT_DIRECTORY" --config scoper.php --force --ansi --working-dir "$BUILD_DIRECTORY";
-else
-    echo "scoping with specify PHP80_BIN_PATH env";
-    $PHP80_BIN_PATH -d memory_limit=-1 php-scoper.phar add-prefix bin config src packages rules vendor composer.json --output-dir "../$RESULT_DIRECTORY" --config scoper.php --force --ansi --working-dir "$BUILD_DIRECTORY";
-fi
-
```

Comment:
New we can use always the PHP 8.1 directly, as php-scoper 0.17.2 supports it, so this can be inlined.